### PR TITLE
Fixing various typos in pharping forms

### PR DESCRIPTION
**Pull request by elephantgloves**
Please review typo fix to labels on various forms.

### CODE REVIEW

File: src/store/reducers/dform/initial/styles.json [hunk 1]
```diff
@@ -545,8 +545,8 @@
                   "label": "PND"
                 },
                 {
-                  "id": "easy_fatigability",
-                  "label": "Easy Fatigability"
+                  "id": "fatigue",
```
Comment (tuttieee) [inline src/store/reducers/dform/initial/styles.json]: All `id` fields of this siblings can be modified (`sob`, `diarrhoea`). Can you please do it?

File: src/store/reducers/dform/initial/styles.json [hunk 2]
```diff
@@ -2864,7 +2864,7 @@
                     },
                     {
                       "field": "pathology.stool.chemical.ph",
-                      "label": "Ph"
+                      "label": "P.H."
```
Comment (tuttieee) [inline src/store/reducers/dform/initial/styles.json]: Is "pH" right (while my original one is also wrong)?
https://en.wikipedia.org/wiki/PH

File: src/store/reducers/dform/initial/styles.json [hunk 3]
```diff
@@ -2920,7 +2920,7 @@
                     },
                     {
                       "field": "pathology.stool.hook_worm.indigested_food_particles",
-                      "label": "Indigested food particles"
+                      "label": "Undigested food particles"
```
Comment (tuttieee) [inline src/store/reducers/dform/initial/styles.json]: `field` can also be modified.

**Comment by tuttieee** [comment]
And you changed the same code both in this PR and https://github.com/asha-nepal/AshaFusionCross/pull/558/files which will cause a conflict when `master` is merged into `release/pharping` in the future if I accept and merge this PR currently.
So, can you please merge `master-add-quotemarks` (a feature branch of https://github.com/asha-nepal/AshaFusionCross/pull/558/files ) into `pharping-typo-fix` (a feature branch of this PR)?

**Comment by tuttieee** [comment]
@elephantgloves And you changed the same code both in this PR and https://github.com/asha-nepal/AshaFusionCross/pull/558/files which will cause a conflict when `master` is merged into `release/pharping` in the future if I accept and merge this PR currently.
So, can you please merge `master-add-quotemarks` (a feature branch of https://github.com/asha-nepal/AshaFusionCross/pull/558/files ) into `pharping-typo-fix` (a feature branch of this PR)?

File: src/store/reducers/dform/initial/styles.json (hunk 1 above)
Comment (elephantgloves) [reply src/store/reducers/dform/initial/styles.json]: Sure thing!

File: src/store/reducers/dform/initial/styles.json (hunk 2 above)
Comment (elephantgloves) [reply src/store/reducers/dform/initial/styles.json]: pH can also be used. Perhaps pH is more common so I will change to pH instead.

File: src/store/reducers/dform/initial/styles.json [hunk 4]
```diff
@@ -1106,7 +1106,7 @@
                 },
                 {
                   "id": "sob",
```
Comment (tuttieee) [inline src/store/reducers/dform/initial/styles.json]: Here is not modified

**Pull request merged**